pkgs/top-level/all-packages.nix

```diff
@@ -9943,7 +9943,16 @@ let
 
   ekho = callPackage ../applications/audio/ekho { };
 
-  electrum = callPackage ../applications/misc/electrum { };
+  electrum = callPackage ../applications/misc/electrum {
```

Comment:
Either put electrum in the python-packages.nix, or pass pythonPackages. Passing explicitly all those packages clutters the definition.